theforeman.org/pipelines/lib/rbenv.groovy

```diff
@@ -0,0 +1,31 @@
+def bundleInstall(version, gemfile=null) {
+    command = "bundle install"
+
+    if (gemfile) {
+        command = "${command} --gemfile=${gemfile}"
+    }
+
+    withRuby(version, "bundle config set path ~/.rubygems")
+    withRuby(version, command)
+}
+
+def bundleExec(version, command, gemfile=null) {
+    command = "bundle exec ${command}"
+
+    if (gemfile) {
+        command = "BUNDLE_GEMFILE=${gemfile} bundle exec ${command}"
+    }
+
+    withRuby(version, command)
+}
+
+def withRuby(version, command) {
```

Comment:
An alternative approach would be to include a check for if a particular environment variable is set (e.g. RUBY_VERSION) then use it unless version is explicitly specified. That would allow not having to pass the ruby version as the first argument everytime if we find that annoying.